### ISSUE TITLE
Can't save

### ISSUE THREAD

**Issue by wjha** [PhilippC/keepass2android#1461]
I'm working with keepass2android for a long time.

Shortly I had to buy a new smartphone and changed all data over. I was working in another country.

Everything seemed to be fine but some time I noticed an error message when it comes to saving a new entry.
Finally I saw, that the "old" data file I transfered was never changed on the new phone. Sadly in that time out I added some entries.

Phone: OnePlus 7 pro  GM1913
IOS: Android 10
Keepass2Android: 1.08b-r1

I found in the logfile an very strange path to "remote content" - //com.ghisler.files/storage ..... 
The URL is from total commander that I use.  But I can't understand how it is placed there. The rest "/storage ..." ist the path I entered in "Umgang mit Dateien" > "Verzeichnis für Dateianhänge"

shall I place some of the log?

**Comment by PhilippC** [comment]
how do you open your database? Did you browse to it via total command and then send it to KP2A from there? Can you please try to select Open file - Android file browser and open from there?

**Comment by wjha** [comment]
Difficult question.
Normally I simply start Keepass, I it dosen't know I use the internal dialog for selecting database. 

What I can clearly say - I first transfer the database to a destiation (i.e. the smartphone) and then use it. Using it together with the total commander? Can't remember.

I just found a way - don't know what happened: When I selected (as before) to save the database there popped up a dialog window what way I want. There I selected the android file browser - and then I could save the file.

Just used Keepass. Tried something, it tried to use the database (middle age name - that I used in August), nearly same mistake. According to logfile:

> 02.11.2020 11:32:05:388 -- Java.Lang.SecurityException: Permission Denial: opening provider com.ghisler.android.TotalCommander.TcContentProviderPlugins from ProcessRecord{14a50fe 6636:keepass2android.keepass2android/u0a172} (pid=6636, uid=10172) that is not exported from UID 10394
  at Java.Interop.JniEnvironment+InstanceMethods.CallNonvirtualObjectMethod (Java.Interop.JniObjectReference instance, Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x0008e] in <71fa3b3620f54526b858e23f7088d717>:0 
  at Java.Interop.JniPeerMembers+JniInstanceMethods.InvokeNonvirtualObjectMethod (System.String encodedMember, Java.Interop.IJavaPeerable self, Java.Interop.JniArgumentValue* parameters) [0x0001f] in <71fa3b3620f54526b858e23f7088d717>:0 
  at Android.Content.ContentResolver.Query (Android.Net.Uri uri, System.String[] projection, System.String selection, System.String[] selectionArgs, System.String sortOrder, Android.OS.CancellationSignal cancellationSignal) [0x000d2] in <5a2a216e90c8491d92d41ca1c79b5274>:0 
  at keepass2android.Io.AndroidContentStorage.IsReadOnly (KeePassLib.Serialization.IOConnectionInfo ioc, keepass2android.Io.OptionalOut`1[T] reason) [0x0003c] in <ecd8a7bc4bb24c4d96e8d9b50a750eff>:0 
-----------
OK, I got my data, synchonized it on PC. 

How to get rid of that strange stuff about ghisler?

**Comment by PhilippC** [comment]
it seems like you opened the file through total commander, so that app provided access to the file. However, this access does not seem to be permanent, i.e. after a while there is an error when opening the file. If you sync with PC, I suggest you move the file to Dropbox (or GDrive, WebDav...) and open if from there with the built-in protocol support in KP2A.

**Comment by wjha** [comment]
Sorry for beeing late.
No, 
- I could get out of the situation by saving the database in a new file. 
- but then, when I opened the APP again, directing it to the new file (i.e. without total commander) same problem arises.

I deleted the APP and reinstalled. In the moment it seems to work. 

A question: even, when I close the APP and remove from active processes - after re open the APP it still seems to have the DB open, it only requests Quick Unlock.
